lago/subnet_lease.py

```diff
@@ -245,26 +244,53 @@ def _take_lease(self, lease, uuid_path, safe=True):
             'Assigned subnet lease {} to {}'.format(lease.path, uuid_path)
         )
 
-    def release(self, subnet):
+    def list_leases(self, uuid=None):
+        try:
+            lease_files = os.listdir(self._path)
+        except OSError as e:
+            # errno 13 = Permission denied
+            if e.errno == 13:
+                raise LagoSubnetLeaseBadPermissionsException(self.path)
+            # errno 2 = No such file or directory
+            if e.errno == 2:
+                return []
+            else:
+                raise
```

Comment:
Will update. Thinking about it, better not swallow the `e.errno == 2` case, I'll just raise there too.